queries.sql

```diff
@@ -1,3 +1,83 @@
 /*Queries that provide answers to the questions from all projects.*/
+SELECT * FROM animals WHERE NAME LIKE '%mon';
+SELECT * FROM animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-12-31';
+SELECT * FROM animals WHERE neutered = 'true' AND escape_attempts < 3;
+SELECT * FROM animals WHERE name IN ('Agumon', 'Pikachu');
+SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
+SELECT * FROM animals WHERE neutered = true;
+SELECT * FROM animals WHERE name <> 'Gabumon';
+SELECT * FROM animals WHERE weight_kg >= 10.4 and weight_kg <= 17.3;
 
-SELECT * from animals WHERE name = 'Luna';
+
+
+
+BEGIN;
+UPDATE animals
+SET species = 'unspecified'
+SELECT * FROM animals;
+ROLLBACk
+SELECT * FROM animals;
+
+UPDATE animals
+SET species = 'digimon'
+WHERE name LIKE '%mon';
+
+UPDATE animals
+SET species = 'digimon'
+WHERE name LIKE '%mon';
+
```

Comment:
- [ ] Great job working on this milestone. From your screenshot, I see that you have `UPDATE` the animals table by setting the species column to `pokemon` for all animals that don't have species already set but I cannot find the command here in `queries.sql` file. Kind verify this and add the command

![image](https://github.com/ibugithub/Vet-Clinic/assets/62849872/6d45e495-c70d-45b9-b5aa-f53d75eecdbe)